tests/test_throttling.py

```diff
@@ -462,6 +465,48 @@ def test_throttle_raises_error_if_rate_is_missing(self):
         with pytest.raises(ImproperlyConfigured):
             SimpleRateThrottle()
 
+    def test_throttle_raises_error_if_rate_is_incorrect(self):
+        SimpleRateThrottle.rate = 'rate'
+        with pytest.raises(ImproperlyConfigured):
+            SimpleRateThrottle()
+
+        SimpleRateThrottle.rate = 'rate/hour'
+        with pytest.raises(ImproperlyConfigured):
+            SimpleRateThrottle()
+
+        SimpleRateThrottle.rate = '100/foos'
+        with pytest.raises(ImproperlyConfigured):
+            SimpleRateThrottle()
+
+        SimpleRateThrottle.rate = '100/10foos'
```

Comment:
Change the above from `100/century` and `100/10century` to `100/foos` and `100/10foos` to handle cases where the time character can be found in the string ("s" in this case)